src/include/simeng/arch/aarch64/Instruction.hh

```diff
@@ -224,6 +224,7 @@ enum class InstructionException {
   EncodingUnallocated,
   EncodingNotYetImplemented,
   ExecutionNotYetImplemented,
+  AliasNotYetImplemented,
```

Comment:
Could you ensure there are tests implemented to ensure this exception gets triggered correctly with the expected result / console printout

Reply:
Resolving as implementing this is being done as part of PR #352